source/elements/oneDAL/source/introduction/building-blocks.rst

```diff
@@ -0,0 +1,4 @@
+Predict&Traing - Building blocks
```

Comment:
Training&Prediction would be more correct.
But even like this it's not clear what this section is about. Why not just "Building blocks"? It seems that the focus is on them.

Reply:
Fixed